managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -2410,7 +2410,7 @@ void internalTrimLedgers(boolean isTruncate, CompletableFuture<?> promise) {
                     if (log.isDebugEnabled()) {
                         log.debug("[{}] Ledger {} not deleted. Neither expired nor over-quota", name, ls.getLedgerId());
                     }
-                    break;
```

Comment:
why are we removing this `break` ? this way the loop will continue to run

Reply:
because we need to close the ReadHandle, otherwise, only one ReadHandle will be closed.